simtools/simtel/simtel_histograms.py

```diff
@@ -120,28 +116,45 @@ def _check_consistency(self, first_hist_file, second_hist_file):
                 self._logger.error(msg)
                 raise InconsistentHistogramFormat(msg)
 
-    def combine_histogram_files(self):
+    @property
+    def combined_hists(self):
         """Add the values of the same type of histogram from the various lists into a single
         histogram list."""
         # Processing and combining histograms from multiple files
-        self.combined_hists = []
-        n_files = 0
-        for hists_one_file in self.list_of_histograms:
-            count_file = True
+        if self._combined_hists is None:
+            self._combined_hists = []
+            n_files = 0
```

Comment:
Can this counting here maybe be simplified and instead using the length of the list of histograms?

Reply:
Indeed, I simplified it now.